GrandCentralBoard/App/Info.plist

```diff
@@ -35,6 +35,10 @@
 	</dict>
 	<key>UIMainStoryboardFile</key>
 	<string>Main</string>
+	<key>UIAppFonts</key>
+	<array>
+		<string>Climacons.ttf</string>
```

Comment:
It's an [open-source font](http://adamwhitcroft.com/climacons/), the terms explicitly state that they may be used in any personal or commercial use (at the bottom of the page).